src/components/ShareTrajectoryModal/style.css

```diff
@@ -25,28 +40,58 @@
     width: 395px;
 }
 
-.copy-button {
+.embed-header {
     display: flex;
+    justify-content: space-between;
+}
+
+.settings-button {
+    border: none;
+    background-color: (var(--light-theme-bg));
+    font-size: 16px;
+    cursor: pointer;
+}
+
+.embed-settings {
+    margin-bottom: 10px;
+}
+
+.accent-text {
+    font-size: 16px;
```

Comment:
are all these font-size: 16 s necessary? Same question for the font-weight

Reply:
Was able to remove a few of these size and weight rules